sdk/cosmos/azure-cosmos/azure/cosmos/http_constants.py

```diff
@@ -379,6 +379,7 @@ class SubStatusCodes(object):
     REDUNDANT_COLLECTION_PUT = 1009
     SHARED_THROUGHPUT_DATABASE_QUOTA_EXCEEDED = 1010
     SHARED_THROUGHPUT_OFFER_GROW_NOT_NEEDED = 1011
+    AAD_REQUEST_NOT_AUTHORIZED = 5300
```

Comment:
where this is used?

Reply:
it's not used but I figured keeping track of it within the SDK would be useful in case we do need it, pretty sure we don't directly use a lot of the other ones either